lib/core/src/lib/info-drawer/info-drawer.component.ts

```diff
@@ -79,4 +88,45 @@ export class InfoDrawerComponent {
     onTabChange(event: MatTabChangeEvent) {
         this.currentTab.emit(event.index);
     }
+
+    getInfoDrawerIcon(node: Node) {
+        if( node.isFolder ){
+          return this.getFolderIcon(node);
+        }
+        if( node.isFile ){
+          return this.thumbnailService.getMimeTypeIcon(node.content.mimeType);
+        }
+        return this.thumbnailService.getDefaultMimeTypeIcon();
+      }
+
+      private getFolderIcon(node: Node) {
+          if (this.isSmartFolder(node)) {
+            return this.thumbnailService.getMimeTypeIcon('smartFolder');
+          } else if (this.isRuleFolder(node)) {
+            return this.thumbnailService.getMimeTypeIcon('ruleFolder');
+          } else if (this.isALinkFolder(node)) {
+            return this.thumbnailService.getMimeTypeIcon('linkFolder');
+          } else {
+            return this.thumbnailService.getMimeTypeIcon('folder');
+          }
+      }
+
+      isSmartFolder(node: Node) {
+        const nodeAspects = this.getNodeAspectNames(node);
+        return nodeAspects.indexOf('smf:customConfigSmartFolder') > -1 || nodeAspects.indexOf('smf:systemConfigSmartFolder') > -1;
+      }
+
+      isRuleFolder(node: Node) {
+        const nodeAspects = this.getNodeAspectNames(node);
+        return nodeAspects.indexOf('rule:rules') > -1 || nodeAspects.indexOf('rule:rules') > -1;
+      }
+
+      isALinkFolder(node: Node) {
```

Comment:
change from isALinkFolder to isLinkFolder() method

Reply:
Updated